src/cluster-config/editor.ts

```diff
@@ -0,0 +1,40 @@
+import { EditorNodeDef, EditorNodeProperties } from 'node-red';
+
+export interface ClusterConfigEditorProperties extends EditorNodeProperties {
+    clusterName: string;
+}
+
+const ClusterConfigEditor: EditorNodeDef<ClusterConfigEditorProperties> = {
+    category: 'config',
+    color: '#a6bbcf',
+    defaults: {
+        name: {value:""},
+        clusterName: {value: ""},
+    },
+    inputs:0,
+    outputs:0,
+    icon: "file.png",
+    oneditsave: Save,
+    oneditprepare: Restore,
+    label: function() {
+        return this.name||"cluster-config";
+    }
+}
+
+export default ClusterConfigEditor;
+
+function Save() {
+    var property = $("#node-input-name")
+    this.name = property.val();
+
+    var property = $("#node-input-cluster-name")
+    this.clusterName = property.val();
+}
+
+function Restore() {
+    var property = $("#node-input-name")
+    property.val(this.name);
+
+    var property = $("#node-input-cluster-name")
+    property.val(this.clusterName);
+}
```

Comment:
For config node types editor does not resolve inputs from html tags. And not sure if this is node bug or expected behaviour. I have to manually get and restore those. This was/is such a bit time-sink that I would like to with current way and potentially revisit this. 

Maybe issue?